### PR TITLE
ID Name and Guest Pass Terminal Tweaks

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR tweaks how ID's (and PDA's) show their name when examined by removing the superfluous brackets.

## changes
- makes identification card's names flow better when read. i.e. `ID card - Bob Jones, Janitor (NT)` instead of `ID card (Bob Jones (Janitor (NT)))`.
- cleans up and improves the guest pass terminal's code.
- makes the arrivals announcer announce the map type instead of just "station".

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

File: code/modules/modular_computers/hardware/card_slot.dm [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-								   /obj/item/lipstick,
-								   /obj/item/device/flashlight/pen,
-								   /obj/item/clothing/mask/smokable/cigarette
-								   )
-	var/obj/item/stored_item //Used for pen, crayon, and lipstick insertion/removal
+	// Storing Items
+	var/list/allowed_items = list(
+		/obj/item/pen
+	)
+	var/obj/item/stored_item // Used for pen insertion and removal.
```
Comment (alsoandanswer) [inline code/modules/modular_computers/hardware/card_slot.dm:16]: Why are you removing this?

Comment (SleepyGemmy) [reply code/modules/modular_computers/hardware/card_slot.dm:16]: feature is still there.

Comment (alsoandanswer) [reply code/modules/modular_computers/hardware/card_slot.dm:16]: As in, allowing to store things other than pens.

**Comment by alsoandanswer** [comment]
please re-add the ability to store things other than pens in PDAs. this isnt a bug nor an oversight, i'd know because i personally added this feature. just put it back and ill approve the pr

File: code/modules/modular_computers/hardware/card_slot.dm [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-	var/list/allowed_items = list(/obj/item/pen,
-								   /obj/item/lipstick,
-								   /obj/item/device/flashlight/pen,
-								   /obj/item/clothing/mask/smokable/cigarette
-								   )
-	var/obj/item/stored_item //Used for pen, crayon, and lipstick insertion/removal
+	// Storing Items
+	var/list/allowed_items = list(
+		/obj/item/pen
+	)
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/hardware/card_slot.dm:15]: Readd the other objects. Unsure why you removed those.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/hardware/card_slot.dm (hunk 2 above)
Comment (SleepyGemmy) [reply code/modules/modular_computers/hardware/card_slot.dm]: addressed.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**